src/main/webapp/js/build.js

```diff
@@ -7,11 +7,11 @@ if (rebuildButton) {
     if (rebuildButton.dataset.parameterized === 'true') {
       window.location.href = buildUrl
     } else {
+      const headers = {}
```

Comment:
Ah there’s a JavaScript API for this search fetch in Jenkins core, it can create the header for

Reply:
Know roughly what it is off the top of your head? I'll take a look for it too.

Reply:
Never mind, I think I found it. Will try using it.